backend/index.js

```diff
@@ -1,6 +1,28 @@
 let devTool;
 let rootNode;
 let rdtOnCommitFiberRoot;
+let updatedFiberTree;
+
+// Listen for clicks
```

Comment:
TO DO: Event listener is still buggy; sends the fiber tree on the second click not first